doc/source/plasma-object-store.rst

```diff
@@ -0,0 +1,43 @@
+The Plasma Object Store
+=======================
+
+Plasma is a high-performance shared memory object store originally developed in
+Ray and now being developed in `Apache Arrow`_. See the `relevant
+documentation`_.
+
+Using Plasma with Huge Pages
+----------------------------
+
+On Linux, it is possible to increase the write throughput of the Plasma object
+store by using huge pages. You first need to create a file system and activate
+huge pages as follows.
```

Comment:
I would say "for example, as follows". There are other ways. I actually now prefer to use hugeadm, if it's installed.